ynab.gemspec

```diff
@@ -25,7 +25,8 @@ Gem::Specification.new do |s|
   s.summary     = "YNAB API Endpoints Ruby Gem"
   s.description = "Ruby gem wrapper for the YNAB API. Read the documentation at https://api.ynab.com"
   s.license     = "Apache-2.0"
-  s.required_ruby_version = ">= 2.7"
+  s.required_ruby_version = ">= 3.3"
```

Comment:
Do you think it'd be worth adding a note in the README letting users know they can use 2.x version of this client if they are still on Ruby 2?

Reply:
Yes, I think that's a good idea. Done in 2e5e58ae189f36018760d9d43d5c27bd7286b759